.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish new Docker image to GHCR
+on:
+  push:
+    branches:
+      - master
```

Comment:
WDYT about trigger to be only `workflow_dispatch`? I think creating a new image on every push to `master` is not necessary. Manual trigger should be enough for this use case.